wiki/HOF_Framework/publishing-hof-npm.md

```diff
@@ -12,6 +12,19 @@ Sometimes a developer may need to upgrade the HOF framework as an improvement or
 For the correct permissions ensure you are a member of hof organisation on https://www.npmjs.com/.
 You may also need to login to the terminal with your npm credentials before being able to publish.
 
+step 1 : npm login
+it will request for your username and password .
+step 2 : if you have a 403 forbidden message, copy this link and replace <username> with your username https://registry.npmmirror.com/-/user/org.couchdb.user:<username>.
```

Comment:
can you explain what step 2 is doing please?

it seems like it will fix the 403 forbidden message, but with my username it looks like it just returns this in a web page
`{
  "_id": "org.couchdb.user:<myusername>",
  "name": "<myusername>"
}`

Reply:
Yes it will fix the 403 page and return a payload which i believe authorise the login details . i  had this issues then reading via the terminal while login that url was generated with my username . It has much to do with Authorization

Reply:
So did you have to run step 2 and then try step 1 again before it worked?